plugins/pipelines/admins/get.js

```diff
@@ -20,14 +21,28 @@ module.exports = () => ({
 
         handler: async (request, h) => {
             const factory = request.server.app.pipelineFactory;
+            const { scmContext, includeUserToken } = request.query;
```

Comment:
Does this mean that anyone with a  user or Pipeline APItoken can access amn admin users  token ? Sounds like a security hole ?

Reply:
Added check to only allow user with `admin` scope to request for user token

Reply:
It should only be `scope: ['admin', '!guest']`